components/automate-backend-elasticsearch/habitat/plan.sh

```diff
@@ -1,5 +1,5 @@
 # shellcheck disable=SC2148
-UPSTREAM_PKG_IDENT="chef/elasticsearch-odfe/0.10.1.2/20200902162328"
+UPSTREAM_PKG_IDENT="chef/elasticsearch-odfe/0.10.1.2"
```

Comment:
Shouldn't we pin the version?

Reply:
When I first started this task, I was getting dependency issues in the stable channel because their internal packages were pointing to the old version, so it was necessary to release one more build to resolve dependency issues in a stable channel, by this release A2ha team started getting dependency issues in stable2021-q2 and they have to ping the version. As we are moving to a stable channel we don't require the ping version.